pkg/template/template.go

```diff
@@ -0,0 +1,25 @@
+package template
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+)
+
+func Parse(name string, contents string, templateData any) (string, error) {
```

Comment:
This is definitely going to make development easier. With this change, I can slap a debugger on my test and look at the applied template rather than having to crack open the file.